full-parent-jakarta/pom.xml

```diff
@@ -35,8 +35,8 @@
         <version.assertj>3.21.0</version.assertj>
         <version.awaitility>4.1.1</version.awaitility>
         <version.org.jboss.arquillian>1.7.0.Alpha10</version.org.jboss.arquillian>
-        <version.org.jboss.arquillian.container.weld-embedded>3.0.0.Final</version.org.jboss.arquillian.container.weld-embedded>
-        <version.org.jboss.arquillian.wildfly>4.0.0.Alpha1</version.org.jboss.arquillian.wildfly>
+        <version.org.jboss.arquillian.container.weld-embedded>3.0.1.Final</version.org.jboss.arquillian.container.weld-embedded>
+        <version.org.jboss.arquillian.wildfly>5.0.0.Alpha2</version.org.jboss.arquillian.wildfly>
```

Comment:
Note this will require a day to resolve as I'm just releasing this into JBoss Nexus.